README.md

```diff
@@ -9,7 +9,7 @@ It works by generating a presigned URL that the Git LFS client can use to upload
 Git LFS S3 is available on RubyGems.
 
 ``` bash
-gem install git-lfs-s3
+gem install lsst-git-lfs-s3
```

Comment:
I agree. I'm going to remove all the objectionable bits in a branch and push that back upstream.